$OpenBSD: patch-packages_ssl_ssl4pl_c,v 1.1 2012/10/01 21:16:00 edd Exp $

On linux, there is a global variable called "timezone", whereas
we have a struct member on OpenBSD.

--- packages/ssl/ssl4pl.c.orig	Tue Oct 22 11:48:46 2013
+++ packages/ssl/ssl4pl.c	Thu Dec  5 10:37:00 2013
@@ -483,7 +483,7 @@ unify_asn1_time(term_t term, ASN1_TIME *time)
   /* mktime assumes that the time_tm contains information for localtime. Convert back to UTC: */
   if ((time_t)-1 != result)
   { result += lSecondsFromUTC; /* Add in the UTC offset of the original value */
-    result -= timezone; /* Adjust for localtime */
+    result -= time_tm.tm_gmtoff; /* Adjust for localtime */
   } else
   { ssl_deb(2, "mktime() failed");
     return FALSE;
